components/backdrop/backdrop.js

```diff
@@ -204,21 +196,12 @@ export function preventBodyScroll() {
 function showAccessible(elems) {
 	for (let i = 0; i < elems.length; i++) {
 		const elem = elems[i];
-		const role = elem.getAttribute(BACKDROP_ROLE);
-		if (role) {
-			elem.setAttribute('role', role);
-			elem.removeAttribute(BACKDROP_ROLE);
+		const ariaHidden = elem.getAttribute(BACKDROP_ARIA_HIDDEN);
+		if (ariaHidden) {
+			elem.setAttribute('aria-hidden', ariaHidden);
```

Comment:
I personally always prefer `hasAttribute` for this kind of thing, but this matches what the old code was doing so 🤷 .

```suggestion
		if (elem.hasAttribute(BACKDROP_ARIA_HIDDEN)) {
			elem.setAttribute('aria-hidden', elem.getAttribute(BACKDROP_ARIA_HIDDEN));
```